tcf_website/models/models.py

```diff
@@ -142,25 +215,30 @@ def reviews(self):
 
 
 class Instructor(models.Model):
-    """Instructor model.
+    """Instructor model. Represents an instructor, such as Louis Bloomfield or Kenneth Elzinga.
 
-    Belongs to many departments.
-    Has many courses.
-    Has many departments.
+    Considerations:
+    An instructor doesn't have any Courses. Instead, it has a collection of Section objects,
```

Comment:
A few more points of context about Section:

- `Section` objects link `Course`, `Instructor`, and `Semester`. It helps make sure there are no duplicates, like a db relationship. If you look at the docstring it says:
    ```
    Section model.

    Belongs to a Course.
    Has many Instructors.
    Has a Semester.
    ```
    - So I'm not sure if Instructors can have Sections if Sections have Instructors...
- `Section` objects are created when new course data is loaded in `tcf_website/management/commands/load_semester.py`
- `Review` objects, however, do not contain `Section` foreignkeys (though it might make sense). They have `Course` and `Instructor` foreignkeys to make filtering easier.
- We’re not using sections to do “autocomplete” on review forms, either. I'm not sure what else Section is currently used for other than ensuring no duplicates. Could see some uses for it in the future like adding timeslot/room fields, class size/waiting list, etc.

Reply:
Thanks for the added context. I'll write this into the docstring. 

> So I'm not sure if Instructors can have Sections if Sections have Instructors...

I see why having this relationship reduplicated in the docs could be confusing, but my intent was to actually make reading `Instructor` easier. If you're looking at the model code for `Section`, you can see the `ManyToManyField` linking it to `Instructor`s—easy enough to infer the relationship.

But because `Instructor` is part of this many-to-many relationship without having `ManyToManyField` declared in its model, it's not obvious that `Instructor` has this relationship. In my view, the docstring for a model should help you understand all of its relationships to other models. This will introduce redundancy, and I think that's a good thing.

It should also be noted that this kind of relationship redundancy was present in the docs before this PR. For example, the docstring for `School` says it "has many Departments" while `Department` "belongs to a school," even though the relationship is actually declared through a `ForeignKey` in `Department`. Or the many-to-many relationship between `Instructor`s, which "belong to many departments," and `Department`s, which have "many Instructors."